db/migrate/20200130160919_contextualize_identifier_schemes.rb

```diff
@@ -0,0 +1,8 @@
+class ContextualizeIdentifierSchemes < ActiveRecord::Migration
+  def change
+    add_column :identifier_schemes, :for_auth, :boolean, default: false
+    add_column :identifier_schemes, :for_orgs, :boolean, default: false
+    add_column :identifier_schemes, :for_plans, :boolean, default: false
+    add_column :identifier_schemes, :for_users, :boolean, default: false
```

Comment:
I forget if these field names were needed for the model polymorphism.
If not, we might want to condiser bitflags (if not mutually exclusive), or an enum (if they are)

Reply:
yes, a bit flag would work here